src/bitcoin-cli.cpp

```diff
@@ -456,6 +458,16 @@ class NetinfoRequestHandler : public BaseRequestHandler
         if (conn_type == "addr-fetch") return "addr";
         return "";
     }
+    std::string FormatServices(const UniValue& services)
+    {
+        std::string str;
+        for (size_t i = 0; i < services.size(); ++i) {
+            if (services[i].isNull()) continue;
```

Comment:
Do we expect to receive a null in `servicesnames` among other entries?
If we don't, maybe this should be something like `assert(!services[i].isNull());` so the issue can be noticed.

Reply:
Sorry, I wasn't as clear as I could have been.  It seems possible to have no services (e.g. `servicesnames` has no elements, in which case the for loop wouldn't be entered, right?).  What I meant to ask is if we think we'll see `servicesnames` with some non-zero number of elements, with at least one element satisfying the check to `isNull()`?  In other words, does the `continue` ever actually happen?

Reply:
Looking at it more, I don't think it's needed and won't be hit, so removed that line for now. (Thanks!)